youtube_dl/extractor/adobepass.py

```diff
@@ -1492,21 +1496,44 @@ def extract_redirect_url(html, url=None, fatal=False):
                             'Content-Type': 'application/x-www-form-urlencoded'
                         })
                 else:
-                    # Some providers (e.g. DIRECTV NOW) have another meta refresh
-                    # based redirect that should be followed.
-                    provider_redirect_page, urlh = provider_redirect_page_res
-                    provider_refresh_redirect_url = extract_redirect_url(
-                        provider_redirect_page, url=urlh.geturl())
-                    if provider_refresh_redirect_url:
-                        provider_redirect_page_res = self._download_webpage_handle(
-                            provider_refresh_redirect_url, video_id,
-                            'Downloading Provider Redirect Page (meta refresh)')
-                    provider_login_page_res = post_form(
-                        provider_redirect_page_res, self._DOWNLOADING_LOGIN_PAGE)
-                    mvpd_confirm_page_res = post_form(provider_login_page_res, 'Logging in', {
-                        mso_info.get('username_field', 'username'): username,
-                        mso_info.get('password_field', 'password'): password,
-                    })
+                    if mso_id == 'Fubo':
```

Comment:
Looks like you have a unnecessarily nested if block. You should deindent everything you've changed, change the first if to an elif and remove the else above it.